tests/integration/test_agent.py

```diff
@@ -29,7 +29,7 @@
     workspace_base=os.getenv('WORKSPACE_BASE'),
     workspace_mount_path=os.getenv('WORKSPACE_MOUNT_PATH'),
     sandbox=SandboxConfig(
-        use_host_network=True,
+        use_host_network=False,
```

Comment:
What if we remove this line instead, shouldn't an env or config.toml value be picked up then, iirc? 🤔 
And for GitHub CI we then can set the env var in the workflow.

Reply:
I think it's better to make the tests deterministic here--everyone should get the same result regardless of what's in their config.toml

Reply:
It's not about that, it's about different platforms in combination with docker can result differently based on this setting. If turned off, I fear that these will break in WSL, for example.

Reply:
But isn't this outside of the scope of the tests, as to me it is a platform issue and is not part of the test results.
The user should have the option to run the tests with the option being True or False.